exampleSite/theme/inventory

```diff
@@ -0,0 +1 @@
+../..
```

Comment:
It looks like this was changed from a [symbolic link](https://www.ibm.com/support/pages/explanation-symbolic-link-and-how-create-it) into a file that contains the path back to the theme. This is not quite what we want to happen, since it needs to be a symbolic link in order for the [example site](https://unicef.github.io/inventory-hugo-theme/) to build and deploy successfully.

Normally you would have seen this feedback in the Pull Request, but I noticed the _Build forked pull requests_ feature was disabled. I enabled it now, so for the next commit you push, it should trigger a CircleCI build which will tell you whether your changes build successfully or not:

![Screenshot of the CircleCI configuration page for the unicef/inventory-hugo-theme repo, with the setting "Build forked pull requests" enabled](https://user-images.githubusercontent.com/4721034/153934694-157128ef-a2e3-47f4-a55f-d8e63bf78bb3.png "Screenshot of the CircleCI configuration page for the unicef/inventory-hugo-theme repo, with the setting \"Build forked pull requests\" enabled")

Could you please undo this change and turn it back into a symbolic link? How to do this will depend on what operating system you are running (e.g. Linux, Windows, macOS).